OpenRobertaRobot/src/main/java/de/fhg/iais/roberta/transformer/Jaxb2ConfigurationAstHelper.java

```diff
@@ -81,9 +83,26 @@ public static Configuration block2OldConfigurationWithFixedBase(
         for ( Value value : values ) {
             String portName = value.getName();
             String userDefinedName = portName.substring(1);
-            boolean isActor = !portName.startsWith(sensorsPrefix);
+            ConfigurationCategory componentCategory;
+            if(portName.contains("AST_"))
+            {	componentCategory = ConfigurationCategory.ASSET;
```

Comment:
please reformat the code. For eclipse see: https://github.com/OpenRoberta/openroberta-lab/wiki/Importing-into-Eclipse#add-a-custom-code-style-formatter - this should also work for intellij.